nextflow/nextflow_examples.ipynb

```diff
@@ -0,0 +1,1169 @@
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9ae4e202-652e-4197-a2c6-a466d1be9ac9",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "!nextflow log"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e8257646-3704-4d75-9af0-141bf5266a89",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "### Optional: Resume pipeline execution from cached data\n",
+    "\n",
+    "Nextflow pipelines can be run with the [`--resume` flag](https://www.nextflow.io/docs/latest/getstarted.html#modify-and-resume). This flag causes the pipeline run to rely on existing, cached intermediate results for some tasks; computation of new intermediate results is performed only where inputs or the pipeline have been modified. This feature allows greater efficiency and speed in pipeline development and debugging.\n",
+    "\n",
+    "To resume pipeline execution for a specific run, provide the session ID of the desired run following the `--resume` flag in the cell below (session ID should be output by the `nextflow log` command in previous cell). If no modifications have been made to the pipeline or its configuration, resuming a successful run should take about 20 minutes. \n",
+    "\n",
+    "\n",
+    "<div class=\"alert alert-block alert-success\">\n",
+    "<b>Note:</b> \n",
+    " The output directory in the command to resume workflow execution (below) differs from the output directory in the original command to launch the workflow. This a necesssary workaround due to a <a href=\"https://github.com/nextflow-io/nextflow/issues/1189https://github.com/nextflow-io/nextflow/issues/1189\">known Nextflow bug</a> in which an error is thrown in instances where an output file of the same name already exists in the output directory (due to the previous run executing successfully and publishing the MultiQC report to the expected output path). If the workflow run you are resuming did not succeed, you may change the output directory to the original destination in the command below.\n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a48fdef8-2140-40fe-9686-37fd67d02285",
+   "metadata": {
+    "tags": [
+     "\"output_scroll\""
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "!terra nextflow -c /home/jupyter/nextflow.config run nf-core/rnaseq -resume <SESSION_ID> -profile google -w $(terra resource resolve --name=ws_files_autodelete_after_two_weeks)/nf_core/wd -r 3.11.1 \\\n",
```

Comment:
This needs the `gls` profile instead here as well. (same as with the initial run)

Reply:
Fixed.